app/main/presenters/search_results.py

```diff
@@ -27,3 +27,13 @@ def _add_highlighting(self, service):
                 service['serviceSummary'] = Markup(
                     ''.join(service['highlight']['serviceSummary'])
                 )
+
+
+class AggregationResults(object):
+    """Provides access to the aggregation results information"""
+
+    def __init__(self, response):
+        self.results = response['aggregations']
+        self.total = response['meta']['total']
+        if 'page' in response['meta']['query']:
+            self.page = response['meta']['query']['page']
```

Comment:
If we're making objects, I'd prefer them to have a consistent interface - in this case, having `page=None` rather than not having that field at all.

So in this case maybe something like `self.page=response['meta']['query'].get('page')`?